partitura/utils/globals.py

```diff
@@ -192,6 +192,24 @@
     {"type": "long", "dots": 3},
 ]
 
+STRAIGHT_DURS = np.array(
```

Comment:
I would add a comment with a high level explanation about why this is useful! Now it looks a bit obscure.